Microsoft.PowerShell.Archive/Microsoft.PowerShell.Archive.psm1

```
@@ -25,7 +25,7 @@
 '@
 }
 
-Import-LocalizedData  LocalizedData -filename ArchiveResources
+Import-LocalizedData LocalizedData -filename ArchiveResources -UICulture "en-US"

```

You already have the En-US text embeded above ... don't we just want Import-LocalizedData to fail silently?

That way, if someone adds a localization, you pick it up, otherwise you continue with the embedded data...

---

100% agreed. As I've stated in the original issue, this is an Import-LocalizedData bug. This will hit any number of modules it has nothing to do with Archive itself. If you don't want Import-LocalizedData then at most it should throw a warning.